keras/backend/theano_backend.py

```diff
@@ -584,6 +584,11 @@ def softplus(x):
     return T.nnet.softplus(x)
 
 
+def softsign(x):
+    # return T.nnet.softmax(x)
```

Comment:
I saw that for softmax, theano was only used in the 2D case, and not being at all sure of what I am doing I preferred to implement it directly, and leave the comment as a reminder of something that could be optimised later.